components/ParameterForm/ParameterForm.vue

```diff
@@ -8,6 +8,21 @@
       novalidate
       @submit.prevent="submitForm"
     >
+      <template v-if="submitError">
+        <CAlert color="danger">
+          <CAlertHeading>
+            <CIcon icon="cilWarning" class="flex-shrink-0 me-2" width="24" height="24" />
+            Error
+          </CAlertHeading>
```

Comment:
There's an argument to made for pulling out the common error display stuff into a component, but I'd be happy for that refactor to be done after the workshop if you make a ticket for it!

Reply:
jidea-319 now exists